Organization/Organization+CoreDataProperties.m

```diff
@@ -0,0 +1,51 @@
+#import "Organization+CoreDataProperties.h"
+#import "Employee+CoreDataClass.h"
+
+@implementation Organization (CoreDataProperties)
+
++ (NSFetchRequest<Organization *> *)fetchRequest {
+	return [[NSFetchRequest alloc] initWithEntityName:@"Organization"];
+}
+
+- (void)addEmplsObject:(Employee *)value
+{
+    [self willChangeValueForKey:@"empls"];
```

Comment:
Why are you doing this way, if you have `- (void)addEmplsObject:(Employee *)value;` generated method?